Sources/Mantis/CropViewConfig.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit
+
+public struct CropViewConfig {
+    public var cropViewMinimumBoxSize: CGFloat = 42
```

Comment:
Maybe? Since it's now part of `CropViewConfig`

```suggestion
    public var minimumBoxSize: CGFloat = 42
```

Also what exactly is this? Is the box size the thing we also control by the new properties?

Reply:
It is used to limit the minimum crop box size when updating crop box's frame. It is probably better to just keep it internal rather than public. I will think about it again.